config/test.json

```diff
@@ -0,0 +1,9 @@
+{
+	"sanity": {
+		"dataset": "test",
+		"token": "skEmSeVTBs1vhTwhTlib08u5Gw6TGgZmkCYCXBtVjQVvzVZF0lMJQCpU2LVwQNVBln9GBYvdDUQKnKNHGhMsbanw3MqmPXPK5opspTqwEL29VhtNGRJL65SG1jVI26KGuFUu8KICiKe7a2KKZT0PE8osIystE5BKqntYg7b4QGRjQFfOozOG"
```

Comment:
Token skal generelt ikke legges inn i Github - er litt lat her fordi det bare er for test-data, men bør egentlig fikse det :p